### ISSUE TITLE
check-vanilla should run `CREATE EXTENSION citus`

### ISSUE THREAD

**Issue by lithp** [citusdata/citus#1460]
check-vanilla runs installcheck and is useful for finding places where we've changed the postgres behavior. However, it misses some bugs which only occur when both the library and the extension are loaded. Some examples: https://github.com/citusdata/citus/issues/1456, `create table test (a int, b date) PARTITION BY RANGE(b);` (we shipped both of these crashes in 6.2)

**Comment by onderkalaci** [comment]
> create table test (a int, b date) PARTITION BY RANGE(b); (we shipped both of these crashes in 6.2)

Just a not, partitioning support comes with pg_10.

Also, +1 for fixing this in this release.

**Comment by lithp** [comment]
Played with this a little and it's not trivial because we add output such as:

```diff
  CREATE ROLE regress_regrole_test;
+ NOTICE:  not propagating CREATE ROLE/USER commands to worker nodes
+ HINT:  Connect to worker nodes directly to manually create all necessary users and roles.
```

**Comment by marcocitus** [comment]
we should probably add a GUC to suppress those messages

**Comment by lithp** [comment]
I simulated the GUC by removing the log statements and (on postgres/master and our support_pg10 branch) received this diff: https://gist.github.com/lithp/fac8e12445bb25e2e0df3040c280f857

**Comment by lithp** [comment]
Although, in lieu of a GUC we could also only emit the messages if there are actually active workers.

**Comment by onderkalaci** [comment]
Fixed by #6018
